nightly_tests/run.sh

```diff
@@ -142,7 +142,9 @@ bash deploy.sh --stack-name "${STACK_NAME}"
 echo "Sleeping for 360s to give enough time for stack to fully come up..."
 sleep 360  # give enough time for stack to fully come up. TODO: revisit this approach
 
-aws cloudformation describe-stack-events --stack-name ${STACK_NAME} >> cloudformation_events.txt
+# Cloud formation smoke_test
+bash smoke_test.sh "${STACK_NAME}" >> cloudformation_events.txt
```

Comment:
I think this line should be moved down below the loop that checks for a non-error return code from the ECS httpd.  This is because inside the smoke test script, an actual endpoint web request test (see my other comments in smoke_test.sh) should be run, and that can't be run until the MC is fully up and accessible.  In other words the smoke test should include a web accessibility test.